google/compute/artman_compute.yaml

```diff
@@ -0,0 +1,24 @@
+common:
+  api_name: compute
+  api_version: v1
+  organization_name: google-cloud
+  gapic_yaml: v1/compute_gapic.yaml
+  # TODO(andrealin): Make discovery_doc a relative path to google/compute
+  discovery_doc: google/compute/v1/compute.v1.json
+artifacts:
+- name: java_discogapic
+  type: DISCOGAPIC
```

Comment:
This is a new type in https://github.com/googleapis/artman/pull/409/files#diff-371ed57b95bd30de3b3331f9cc2ff446R335